src/views/virtualmachines/actions/actions.ts

```diff
@@ -15,8 +15,8 @@ import {
 } from '@openshift-console/dynamic-plugin-sdk';
 
 export const VirtualMachineInstanceMigrationModel: K8sKind = {
-  label: 'Virtual Machine Instance Migration',
-  labelPlural: 'Virtual Machine Instance Migrations',
+  label: 'VirtualMachineInstance Migration',
```

Comment:
since this is a model for a resource it should be `VirtualMachineInstanceMigration` and `VirtualMachineInstanceMigrations`